### PR TITLE
Remove authoring and automated scene view creation

### PR DESCRIPTION
**Pull request by spdracer22**
## Summary
This PR removes authoring and automated scene view creation to simplify this demo/accelerator code. Authoring was not explicitly enabled or exposed by the deployed demo.

## Test Plan
- [ ] Check out PR
- [ ] `yarn install`
- [ ] `yarn dev`
- [ ] Browse to `https://localhost:3000`
- [ ] Navigate through the work instructions to ensure proper functionality

### CODE REVIEW

File: src/components/Home.tsx [hunk 1]
```diff
@@ -92,41 +49,45 @@ export function Home({ authoring, vertexEnv }: Configuration): JSX.Element {
...
 
-		setReady(true);
-		setSceneViewId(scene.sceneViewId);
-		if (authoring) await initializeScene({ instructions, viewer: v });
+		setIsSceneReady(true);
 	}
 
 	async function onInstructionStepSelected(num: number): Promise<void> {
-		if (!ready) return;
+		console.log('onInstructionStepSelected');
```
Comment (amvertex) [inline src/components/Home.tsx:56]: This can probably be removed

File: src/lib/config.ts [hunk 2]
```diff
@@ -11,22 +12,12 @@ export interface StreamCredentials {
...
-	authoring: envVar('AUTHORING', 'false') === 'true',
 	vertexEnv: envVar('VERTEX_ENV', 'platprod') as Environment,
 };
 
-// Super Car
 export const Credentials: StreamCredentials = {
 	clientId: '08F675C4AACE8C0214362DB5EFD4FACAFA556D463ECA00877CB225157EF58BFA',
 	streamKey: 'wzTRPxHgI8ylWKevQIDPA-GhL_8PhWwqLj4w',
 };
-export const SceneId = '421a79ac-9b12-49b1-b517-4d6760b3c9f8';
```
Comment (amvertex) [inline src/lib/config.ts:23]: Can we leave the scene id as a comment? It's helpful for when we need to update the stream key when the current one expires.

**Review by amvertex — COMMENTED** [review]
Looks great! I only have two small comments.

File: src/components/Home.tsx (hunk 1 above)
Comment (spdracer22) [reply src/components/Home.tsx:56]: 💯 I missed this one

File: src/lib/config.ts (hunk 2 above)
Comment (spdracer22) [reply src/lib/config.ts:23]: Yep

**Review by amvertex — APPROVED** [review]
Looks great!

**Pull request merged**